### PR TITLE
fix: handle publishing messages bigger than max size

### PR DESCRIPTION
**Pull request by kosecki123**
This PR adds the error handling when publishing messages bigger than maximum allowed message size.

### CODE REVIEW

File: specs/schema.yaml [hunk 1]
```diff
@@ -123,7 +123,7 @@ components:
         maxMsgAge:
           type: "number"
           description: "Maximum age of any message in the channel, expressed in nanoseconds."
-          example: 86400000000
+          example: 8640000000000000
```
Comment (kosecki123) [inline specs/schema.yaml:126]: @yartman saw your PR also updates this, I guess safe would be to merge this PR and then yours.

Comment (kosecki123) [reply specs/schema.yaml:126]: btw: this was updated after running schema:generate

Comment (yartman) [reply specs/schema.yaml:126]: sorry, i didn't see this before merge

Comment (kosecki123) [reply specs/schema.yaml]: No problem, solved as it triggered merge conflict.

**Review by yartman — APPROVED** [review]

**Pull request merged**